tests/login.rs

```diff
@@ -7,7 +7,7 @@ extern crate toml;
 use std::io::prelude::*;
```

Comment:
This (and I think at least one other file?) I think changed from 644 permissions to 755, mind reverting back?

Reply:
I will revert those.